app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala

```diff
@@ -46,7 +48,8 @@ final case class WriteStacCatalog(exportId: UUID)(
     try {
```

Comment:
I don't really love how much `IO` is happening here in `try` / `catch` instead of in `IO`, since all the batch jobs I think are now built around `IO`, but it's probably outside of scope for what you're working on here to change that. I opened #5143 to handle that someday.

Reply:
Or the later bit where there's just a massive `foreach` with a bunch of things that might fail